homeassistant/components/device_tracker/ddwrt.py

```diff
@@ -84,12 +86,12 @@ def get_device_name(self, device):
                     return None
 
                 # Remove leading and trailing single quotes.
-                cleaned_str = dhcp_leases.strip().strip('"')
-                elements = cleaned_str.split('","')
-                num_clients = int(len(elements)/5)
+                cleaned_str = dhcp_leases.replace("\'", "").replace(" ", "")
```

Comment:
Looking at the old code, should you not replace double quotes with nothing as well?